panel/widgets/player.py

```diff
@@ -36,19 +37,29 @@ class PlayerBase(Widget):
     show_loop_controls = param.Boolean(default=True, doc="""
         Whether the loop controls radio buttons are shown""")
 
+    show_value = param.Boolean(default=False, doc="""
+        Whether to show the widget value""")
+
     step = param.Integer(default=1, doc="""
         Number of frames to step forward and back by on each event.""")
 
     height = param.Integer(default=80)
 
+    value_location = param.ObjectSelector(
+        objects=["top_left", "top_center", "top_right"], doc="""
+        Location to display the value of the slider
+        ("top_left", "top_center", "top_right")""")
+
```

Comment:
We need to add these new params to the reference docs params intro, and maybe a brief example in one of the cells:

`panel/examples/reference/widgets` for Player.ipynb and DiscretePlayer.ipynb I think.

See https://github.com/holoviz/panel/pull/6087/files for an example